microprofile/WFLY-19792-mp-multiple-wars.adoc

```diff
@@ -0,0 +1,224 @@
+---
+categories:
+- microprofile
+stability-level: default
+# Specify the Feature Development tracker issue for the feature.
+# This must be an issue tracked in https://github.com/orgs/wildfly/projects/7/views/1.
+# To create a Feature Development tracker issue, go to  https://github.com/wildfly/wildfly-proposals/issues/new/choose
+# and select 'Feature Development'
+issue: https://github.com/wildfly/wildfly-proposals/issues/731
+feature-team:
+ developer: kabir
+ sme:
+  - jmesnil
```

Comment:
Should this field enumerate all feature participants since the developer field is not a list but a single item? I find it quote odd that the actual participating SMEs are not listed anywhere?

```suggestion
 sme:
  - jmesnil
  - jamezp
  - jasondlee
  - pferraro
  - rhusar
```